src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color ballColor;
+    private int ballNumber;
+
+    public Color getBallColor() {
+        return ballColor;
+    }
+
+    public void setBallColor(Color ballColor) {
+        this.ballColor = ballColor;
+    }
+
+    public int getBallNumber() {
+        return ballNumber;
+    }
+
+    public void setBallNumber(int ballNumber) {
+        this.ballNumber = ballNumber;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Return a simple concatenating string such as  `"Ball color: " + color + ", ball number: " + number`

[Also, don't forget about this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)